TimeCalc.java

```diff
@@ -1,5 +1,37 @@
 public class TimeCalc {
     public static void main(String[] args) {
-        // Replace this comment with your code
+        String s1 = args[0];
+        int h1 = Integer.parseInt(""+s1.charAt(0)+s1.charAt(1)+"");
+        int m1 = Integer.parseInt(""+ s1.charAt(3)+s1.charAt(4));
+        int addT= Integer.parseInt(args[1]);
+
+        int ttlT= h1 * 60 + m1 + addT;

```

Comment:
The name ttlT is very confusing and not informative, did you mean totalTime? 
In these cases using a variable name that is a bit longer will make your code a lot more readable